docs/specs/build/toc/corners.yml

```diff
@@ -69,14 +68,7 @@ inputs:
   docs/index.md:
 outputs:
   build.log: |
-    [
-      {
-        "message": "Found circular reference: docs/toc.md --> docs/f1/toc.md --> docs/f2/toc.md --> docs/toc.md",
-        "message_severity": "Error",
-        "type": "circular-reference",
-        "file": "docs/toc.md"
-      }
-    ]
+    ["error","circular-reference","Found circular reference: docs/toc.md --> docs/f1/toc.md --> docs/f2/toc.md --> docs/toc.md","docs/toc.md"]
```

Comment:
what's that format?

Reply:
JSON

Reply:
we need this log to generate report, so modeled build.log would be better